### PR TITLE
Couple of functions used for Magic Coat

### PR DESCRIPTION
**Pull request by assidion2**

### CODE REVIEW

File: symbols/overlay29.yml [hunk 1]
```diff
@@ -8641,6 +8641,26 @@ overlay29:
...
         return: Always 0
+    - name: GetExclusiveItemWithEffectFromBag
+      address:
+        EU: 0x23100C8
+      description: |-
+        If an exclusive item with a certain effect is present in the bag and works for the entity, returns true and copies the item struct to the address at item. Otherwise, returns false.
+        
+        r0: Entity pointer
+        r1: exclusive_item_effect_id enum
+        r2: Item pointer [output]
```
Comment (UsernameFodder) [inline symbols/overlay29.yml:8652]:
```suggestion
        r2: [output] Item pointer
```

File: symbols/overlay29.yml [hunk 2]
```diff
@@ -8641,6 +8641,26 @@ overlay29:
         Called when the in-dungeon "others" menu is open. Does not return until the menu is closed.
         
         return: Always 0
+    - name: GetExclusiveItemWithEffectFromBag
```
Comment (UsernameFodder) [inline symbols/overlay29.yml:8644]: Can you fix the symbol ordering? Symbols are ordered by address.

**Review by UsernameFodder — CHANGES_REQUESTED** [review]

File: symbols/overlay29.yml (hunk 2 above)
Comment (assidion2) [reply symbols/overlay29.yml]: Is there a script to automate the ordering?

Comment (UsernameFodder) [reply symbols/overlay29.yml]: Yes there is automation. See [the docs](https://github.com/UsernameFodder/pmdsky-debug/blob/master/symbols/README.md#follow-style-conventions)

**Review by UsernameFodder — CHANGES_REQUESTED** [review]
The [header checks](https://github.com/UsernameFodder/pmdsky-debug/actions/runs/10651869451/job/29524952654?pr=272) are still failing. Can you fix the formatting there as well? Like with the symbols, there's automation; see [the docs](https://github.com/UsernameFodder/pmdsky-debug/blob/master/headers/README.md#follow-style-conventions).

**Comment by assidion2** [comment]
When I try to do make format I get this error: "make: *** No rule to make target `format'.  Stop." I have clang-format installed, and find and xargs seem to also work. Do you know how to solve this?

**Review by UsernameFodder — APPROVED** [review]
Looks good now, thank you!

**Pull request merged**